File: db4on/reference/vb/db4o-reference-chapters/src/com/db4odoc/f1/evaluations/EvenHistoryEvaluation.vb
Imports com.db4o.query
Namespace com.db4odoc.f1.evaluations
    Public Class EvenHistoryEvaluation
        Implements evaluation
        Public Sub Evaluate(ByVal candidate As Candidate) Implements evaluation.Evaluate
            Dim car As Car = DirectCast(candidate.GetObject(), Car)
            candidate.Include(car.History.Count Mod 2 = 0)
        End Sub

    End Class
End Namespace

File: db4on/reference/db4o-reference-chapters/src/com/db4odoc/f1/evaluations/EvenHistoryEvaluation.cs
using com.db4o.query;
	
namespace com.db4odoc.f1.evaluations
{	
	public class EvenHistoryEvaluation : Evaluation
	{
		public void Evaluate(Candidate candidate)
		{
			Car car=(Car)candidate.GetObject();
			candidate.Include(car.History.Count % 2 == 0);
		}
	}
}